model/host/host.go

```diff
@@ -246,6 +246,16 @@ type HostModifyOptions struct {
 	DetachVolume       string
 }
 
+type SpawnHostUsage struct {
+	TotalHosts            int `bson:"total_hosts"`
```

Comment:
maybe some other useful metrics could be:
- total cost (/hour?)
- largest instances
- longest running times
- biggest users (in terms of instance size, running time, etc.)
- number of stopped vs. running

Reply:
I added compute cost since there's an easy metric for that, and number of stopped. For now I think it's fine for this to be overall generals. If at some point we have concerns and want to pinpoint change points (i.e. biggest user) we can revisit.

I also added an instance type set. What I would really like to do is have a map of instance type to count, but I can't figure out how to aggregate this.

Reply:
>What I would really like to do is have a map of instance type to count

would it be possible by adding a separate aggregation on the hosts collections, grouping on instance type?

Reply:
Okay I added something, but I can't figure out how to make it a map without having to loop over it. That being said it'd be a tiny map so this would be fine unless someone knows already how to fix it.